arangod/Aql/ExecutionPlan.cpp

```diff
@@ -2538,7 +2539,7 @@ struct Shower final : public WalkerWorker<ExecutionNode> {
       case ExecutionNode::REMOVE:
       case ExecutionNode::REPLACE:
       case ExecutionNode::UPSERT:
-      case ExecutionNode::MATERIALIZE: {
+      /*case ExecutionNode::MATERIALIZE:*/ {
```

Comment:
Why is that?
```suggestion
      case ExecutionNode::MATERIALIZE: {
```

Reply:
I can delete the comment but `MaterializeNode` is not a `CollectionAccessingNode`.

Reply:
My bad, only the `MaterializeSingleNode` is.